Client/src/main/java/de/baleipzig/pdfextraction/controller/ImportController.java

```diff
@@ -28,9 +51,63 @@ private void continueButtonOnAction() {
         );
     }
 
-    // initialize wird automatisch vom FXML loader aufgerufen wenn die FXML Datei geladen wird
     @FXML
-    private void initialize(){
-        PDFPreview.createPreview(PDFPreviewPlaceholder);
+    private void initialize() {
+
+        // image view resizeble machen
+        AnchorPane parent = (AnchorPane) imageView.getParent();
+        imageView.fitWidthProperty().bind(parent.widthProperty());
+        imageView.fitHeightProperty().bind(parent.heightProperty());
+
+        if (pdfPath == null){
+            pageIndex.setText("Seitenanzahl");
+        } else {
+            updatePdfPreview(PDFPreview.getCurrentPage());
+        }
+    }
+
+    public void onClickPageBack(ActionEvent actionEvent) {
+
+        if (PDFPreview.getCurrentPage() > 1 && pdfPath != null){
```

Comment:
Die Logik zum prüfen ob es eine nächste Seite hat, gehört in den Renderer/Preview